index.html

```diff
@@ -1,111 +1,173 @@
-    <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
 </head>
 <body>
-    <div id="container" class="container">
-        <div class="profile-folder">
-          <div class="folder">
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
-            </div>
-
-            <div class="body">
-              <div class="profile-icon-box">
-                <p id="typeText"></p>
-                <div class="profile-icon">
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon.png" /> <!-- Photo -->
-                </div>
-              </div>
-
-              <div class="name">
-                <p>AYAKO YOKOYA</p> <!-- Name -->
-              </div>
-              <div class="aboutme">
-                <p>Creative Developer <span>(I wanna be...)</span></p> <!-- Profession -->
-              </div>
-
-              <div class="sns-btn-box">
-                <p>My sns accounts are...</p>
-                <a
-                  href="https://github.com/ayakoyokoya/"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  github <!-- Link to social media 1 -->
-                </a>
-                <a
-                  href="https://twitter.com/ayakoyokoya"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  twitter <!-- Link to social media 2 -->
-                </a>
-              </div>
-
-              <div class="portfolio-btn-box">
-                <p>
-                  Do you want to see error? <!-- Any text -->
-                </p>
-                <div class="portfolio">
-                  <button id="error-btn" class="error-btn">
-                    error?
-                  </button>
-                  <a
-                    href="https://ayakoyokoya.github.io/yaba-kawaii-audio-visualizer/dist/"
-                    target="_blank"
-                    class="portfolio-btn"
-                  >
-                    yes! 
-                  </a>
-                </div>
-              </div>
-            </div>
-          </div>
+  <header class="header">
+    <img src="img/Logos.svg" alt="" class="logo">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation item"><a href="" class="nav-link">Home</a></li>
+        <li class="navigation item"><a href="" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Catalog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <button class="search-bar">Search</button>
```

Comment:
@Alinchik1 we have to set the "type" attribute for each button explicitly. If the button sends the form - then type="submit", otherwise type="button"

Reply:
Please do it for each button on your pages

Reply:
fixed